docusaurus.config.js

```diff
@@ -85,13 +83,18 @@ const config = {
             label: 'Community GitHub',
             position: 'right',
           },
+          {
+            href: 'https://discord.gg/dbl',
```

Comment:
Should use a direct invite link instead of the custom URL since it's linked to boosts. (https://discord.gg/EYHTgJX)

Reply:
Sure.